packages/web-app-files/src/helpers/path.js

```diff
@@ -27,6 +27,10 @@ export function getParentPaths(path = '', includeCurrent = false) {
 
   sections.pop()
   while (sections.length > 0) {
+    if (!sections.join('/')) {
```

Comment:
What about changing line 22 to
```js
const sections = s.split('/').filter(Boolean)
```
instead to remove empty strings from the array?

Reply:
That would actually result in a different array at the end. When `''` is removed beforehand, the leading slash would be missing.